statsmodels/genmod/generalized_linear_model.py

```diff
@@ -114,8 +114,6 @@ class GLM(base.LikelihoodModel):
         See Parameters.
     normalized_cov_params : array
```

Comment:
should this also be removed from the docstring?  the attribute is removed below

Reply:
If so, the docstring entry in GLMResults needs to be updated too.

Reply:
Yes that attribute is gone and should also be removed from the docstring.

normalized_cov_params in GLMResults, is the real one evalutated at the final estimate.

Reply:
OK.  This is the kind of thing that I'd be happy to address in a quick-turnaround PR if e.g. @bashtage were anointed to review+merge.

Reply:
A clear docstring fix is usually a fast merge, 
(as long as you don't try to sneak in any other refactoring that might be controversial or need more careful review)

Reply:
> A clear docstring fix is usually a fast merge,

Experience suggests otherwise.  I'll fix this downstream and leave this for the next sucker.